app.js

```diff
@@ -416,9 +461,9 @@ function processModule(moduleName) {
           const args = get(statement, 'expression.arguments');
           if ((args.length === 2) && (args[0].name === 'self') &&
             (args[1].name === 'options')) {
-            const path = get(statement, 'expression.callee.arguments.0.value');
+            const _path = get(statement, 'expression.callee.arguments.0.value');
```

Comment:
Why the underscore? I think we're losing team consistency in use of this naming structure. Val recently used it for an unused parameter, which seems to be a common pattern.

Reply:
Fixed. A very old pattern for me is to use it to avoid a conflict with a global variable when the name is otherwise suitable, but that's not as useful as staying consistent with our use of `_` elsewhere.

Reply:
(pet peeve: the `path` module of nodejs should have been named `paths` back in the dawn times, but they messed it up and we're stuck with it forever. Oh well.)